frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -461,7 +431,17 @@ export function FieldValuesWidgetInner({
             prefix={prefix}
             value={value.filter(v => v != null)}
             onChange={onChange}
-            placeholder={tokenFieldPlaceholder}
+            placeholder={
+              placeholder ??
+              getTokenFieldPlaceholder({
```

Comment:
I don't follow this splitting of placeholder logic. Before all the details were hidden under `getTokenFieldPlaceholder` function. 

It was covered in the function before: `if (placeholder) return placeholder`. 

But I can miss something for sure

Reply:
There's no reason really. This is just moving complexity from one place to another. I don't mind really so I will change it back